README.md

```diff
@@ -40,8 +40,9 @@ rows for example `Cursorfetchone()` or `Cursor.fetchmany()`. By default
 `trino.dbapi.Cursor.arraysize` accordingly.
 
 # Basic Authentication
-The `BasicAuthentication` class can be used to connect to a LDAP-configured Trino
-cluster:
+The `BasicAuthentication` class can be used to connect to a Trino cluster configured with
+the [Password file authentication, LDAP authentication or Salesforce authentication](https://trino.io/docs/current/security/authentication-types.html):
```

Comment:
```suggestion
the [`PASSWORD` authentication type](https://trino.io/docs/current/security/authentication-types.html):
```

Reply:
@hashhar thanks for the suggestions. I guess this issue was started with the idea that just writing PASSWORD authentication does not make it clear what type of authentication can be used hence I added all password authentication types to make it explicit. Do you still want to make it only PASSWORD authentication type?

Reply:
Let's keep it as is then for password auth. Thanks for reminding.